concourse/util.py

```diff
@@ -32,13 +32,18 @@
 
 def list_github_resources(
```

Comment:
hmm.. thinking of it, it might be a good idea to change this function's signature in such a way that it accepts an initialised concourse api instance; this way, there would not be the need to pass-in the version after all..

Reply:
done